controllers/authorization/authorization_controller_test.go

```diff
@@ -140,7 +140,7 @@ var _ = Describe("Checking Authorization Resource Creation", test.EnvTest(), fun
 		}, test.DefaultTimeout, test.DefaultPolling).Should(Succeed())
 	})
 
-	It("should create a PeerAuthentication when a Component is created", func(ctx context.Context) {
+	XIt("should create a PeerAuthentication when a Component is created", func(ctx context.Context) {
```

Comment:
![x](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExbXJzZjFuYzU3czFjdTlrbmh6M2xuZmY2cms0cmoyaGV5am54cmxjcyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/mA76Rv9i2YFFsude5k/giphy.gif)

I think we should just delete this code too?